adaptivefiltering/visualization.py

```diff
@@ -1,9 +1,10 @@
 import IPython
 import ipyvolume.pylab as vis
-import matplotlib.pyplot as plt
+import matplotlib.pyplot as plt, mpld3
```

Comment:
As said, I would prefer to not have mpld3 as an additional dependencies. `ipympl` should be able to handle this.